code/core/src/theming/convert.ts

```diff
@@ -12,10 +12,10 @@ const lightSyntaxColors = {
   red1: '#A31515',
   red2: '#9a050f',
   red3: '#800000',
-  red4: '#ff0000',
+  red4: '#D24400',
```

Comment:
I would remove the color changes from this one. I will probably revamp the syntax highlight color schemes in the nearish future.